SwaggerThemes/SwaggerTheme.cs

```diff
@@ -65,26 +60,8 @@ private static void AddGetEndpoint(WebApplication app, string cssPath, string st
         {
             context.Response.Headers["Cache-Control"] = "public, max-age=3600";
             context.Response.Headers["Expires"] = DateTime.UtcNow.AddDays(2).ToString("R");
-            
             return Results.Content(styleText, "text/css");
```

Comment:
I notice that you removed my ExecuteAsync call. It makes this endpoint more efficient and trim compatible.